AdyenComponents/PayTo/PayToComponent.swift

```diff
@@ -181,12 +227,15 @@ public final class PayToComponent: PaymentComponent,
     }()
 
     /// The identifier picker item.
-    internal lazy var identifierPickerItem: FormStringPickerItem = {
-        let selectableValues = AccountIdentifiers.allCases.map { accountIdentifier in
-            FormStringPickerElement(identifier: accountIdentifier.rawValue, title: accountIdentifier.description)
+    internal lazy var identifierPickerItem: FormContainerItem<FormStringPickerItem> = {
+        let selectableValues = PayIDIdentifier.allCases.map { payIDIdentifier in
```

Comment:
This triggers some questions:
- is it fine to keep enum order, do we need to change it in the future?
- this is not testable, do we need to test it?

Reply:
These are hardcoded values that is reportedly not to change.